src/PayID/xrp-pay-id-client-interface.ts

```diff
@@ -19,3 +21,20 @@ export default interface XRPPayIDClientInterface {
    */
   xrpAddressForPayID(payID: string): Promise<string>
 }
+
+export default interface XrpPayIdClientInterface {
+  /**
+   * @param network The network that addresses will be resolved on.
+   */
+  xrplNetwork: XrplNetwork
+
+  /**
+   * Retrieve the XRP Address associated with a PayID.
+   *
+   * @note The returned value will always be in an X-Address format.
+   *
+   * @param payId The payID to resolve for an address.
```

Comment:
Obviously the parameter `payId` uses camelcase. Do you want to duplicate that with "The payId to ...", or should it be "The PayID to ..."?

Reply:
Yeah, sorry this is a mess. We wrote a *lot* of code before this was all agreed upon. 

In code, we want to camel case things, so we'll always use "PayId" for consistency. This matches up with our other code. 

In docs, we should always call it a "PayID" since that is the noun we've landed on. I'll make this change.